Description: Ignore tests requiring credentials from google
Author: Dong Xu <me@eipi.fun>
Forwarded: no
Reviewed-By: Dong Xu <me@eipi.fun>
Last-Update: 2024-10-04

--- python-google-api-python-client-2.147.0.orig/tests/test_discovery.py
+++ python-google-api-python-client-2.147.0/tests/test_discovery.py
@@ -531,7 +531,7 @@ class DiscoveryErrors(unittest.TestCase)
                 ),
                 static_discovery=False,
             )
-
+    '''
     def test_credentials_and_credentials_file_mutually_exclusive(self):
         with self.assertRaises(google.api_core.exceptions.DuplicateCredentialArgs):
             build(
@@ -543,7 +543,7 @@ class DiscoveryErrors(unittest.TestCase)
                 ),
                 static_discovery=False,
             )
-
+    '''
 
 class DiscoveryFromDocument(unittest.TestCase):
     MOCK_CREDENTIALS = mock.Mock(spec=google.auth.credentials.Credentials)
@@ -2581,7 +2581,7 @@ class Universe(unittest.TestCase):
                         universe_domain=""
                     ),
                 )
-
+        '''
         def test_client_options_universe_configured_with_mtls(self):
             fake_universe = "foo.com"
             discovery = read_datafile("tasks.json")
@@ -2596,7 +2596,7 @@ class Universe(unittest.TestCase):
                             universe_domain=fake_universe
                         ),
                     )
-
+        '''
         def test_client_options_universe_configured_with_api_override(self):
             fake_universe = "foo.com"
             fake_api_endpoint = "https://www.bar.com/"
@@ -2625,7 +2625,7 @@ class Universe(unittest.TestCase):
                         discovery,
                         credentials=credentials,
                     )
-
+        '''
         def test_universe_env_var_configured_with_mtls(self):
             fake_universe = "foo.com"
             discovery = read_datafile("tasks.json")
@@ -2639,7 +2639,7 @@ class Universe(unittest.TestCase):
                     },
                 ):
                     tasks = build_from_document(discovery)
-
+        '''
         def test_universe_env_var_configured_with_api_override(self):
             fake_universe = "foo.com"
             fake_api_endpoint = "https://www.bar.com/"
